docs/root/configuration/cluster_manager/cluster_circuit_breakers.rst

```diff
@@ -7,6 +7,11 @@ Circuit breaking
 * :ref:`v1 API documentation <config_cluster_manager_cluster_circuit_breakers_v1>`.
 * :ref:`v2 API documentation <envoy_api_msg_cluster.CircuitBreakers>`.
 
+.. note::
+
+  Circuit breaking will be completely disabled in the absence of the
```

Comment:
As a top level comment IMO this is a breaking change, because I am sure there are users out there that have not configured circuit breaking for all of their clusters, but potentially depend on the default protection.

Reply:
Yeah we can't do this. This must be opt-in. Stepping back, what is the problem w/ just supplying very large values? I realize it's not optimal, but the flip side is that IMO circuit breaking should not be opt-in, but rather opt-out (we can sympathize that the defaults are arbitrary, but, they are what they are and I don't think we can change them). So I think we should either not do this change at all and have the user supply very large values, or figure out a way in the configuration to make this an optional opt-out.